v2/api/network/customizations/dns_forwarding_ruleset.go

```diff
@@ -23,10 +25,19 @@ func (extension *DnsForwardingRulesetExtension) ClassifyError(
 		return core.CloudErrorDetails{}, err
 	}
 
-	// Override is to treat Conflict as retryable for Redis, if the message contains "try again later"
-	if isRetryableConflict(cloudError) {
+	if isRetryableConflict(cloudError) || isRetryableBadRequest(cloudError) {
```

Comment:
In theory we could remove these `isRetryablerConflict` checks now, since we retry on all conflicts. WDYT @theunrepentantgeek? I can send a separate PR to clean these up.

Reply:
Yes, I think so.

Reply:
Ill do this in a followup PR